src/components/Calculator.jsx

```diff
@@ -0,0 +1,52 @@
+const Calculator = () => {
```

Comment:
- [ ] So far you have done an amazing job with the design :+1: unfortunately it didn't matched the given design as you can see on the screenshots below. kindly adjust layout and colors to match the given design.


| Given Design :+1:  | Your Design |
| --- | --- |
| ![Screenshot from 2023-09-13 19-19-33](https://github.com/Titus-Kiplagat/calculator/assets/63087491/fb703145-1547-4121-93b7-90c4306f18e7) | ![Screenshot from 2023-09-13 19-18-31](https://github.com/Titus-Kiplagat/calculator/assets/63087491/3667f706-d6ec-4a55-98ae-f68490f30a50) |